.github/workflows/build.yml

```diff
@@ -68,7 +68,7 @@ jobs:
 
   build_ios:
     needs: ["test"]
-    runs-on: macos-10.15
+    runs-on: macos-latest
```

Comment:
would it make sense to pin to `macos-11` instead of `latest`?
(also applies for `deploy.yml`)

Reply:
I don't have a strong opinion which one to use, changed it to `macos-11`.

Reply:
Unfortunately, when I changed it to `macos-11`, the Android e2e CI action started to fail. Detox couldn't connect to the Android emulator. The same was happening when I used `macos-12`. Couldn't figure out why, it seems like a house of cards 😢  I had a similar problem with Detox last week, it was doing the same thing when I tried to build with Android SDK 31 or 32. After a couple of days debugging, I found out that it is an open issue in Detox. I think we'll just need to keep it like this and wait for a more stable version of Detox.